src/components/form/FieldSet.tsx

```diff
@@ -0,0 +1,71 @@
+import classNames from 'classnames'
+import { ReactElement, ReactNode } from 'react'
+import { get, useFormState } from 'react-hook-form'
+import { useTheme } from '../../framework'
+import { ClassNameProps, Mode, ModeProps } from '../types'
+import { getCustomLabelCss } from './useCustomInputCss'
+
+export enum FieldSetIndent {
+  none = 'none',
+  all = 'all',
+  label = 'label',
```

Comment:
Wdyt about being more explicit?

```suggestion
  none = 'none',
  labelAndChildren = 'all',
  label = 'label',
```